src/modules/integrator-guide/pages/corteza-studio/index.adoc

```diff
@@ -0,0 +1,122 @@
+
+{PRODUCT_NAME} Studio allows users to customize {PRODUCT_NAME} web applications styling according to their preferences. 
+
+[IMPORTANT]
+====
+For {PRODUCT_NAME} Studio, to function properly, you need to xref:devops-guide:installing-sass/index.adoc[Install dart-sass binaries] in your system and 
+`HTTP_WEBAPP_ENABLED` enviroment variable set to `true`.
+====
+
+== Customizing {PRODUCT_NAME} Web Applications
+
+There are two ways of customizing your {PRODUCT_NAME} web applications style:
+
+=== Branding
+
+Under menu:User Interface[Settings] branding section.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "corteza-studio/branding-section.png",
+    "alias": "corteza-studio-branding-section",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "focus": {
+    "x": 577,
+    "y": 490,
+    "w": 1087,
+    "h": 579
+  },
+  "annotations": [{
+    "kind": "box-note",
+    "x": 375,
+    "y": 212,
+    "w": 1130,
+    "h": 733
+  }]
+}
+----
+
+A screen showing you all of the available {PRODUCT_NAME}-specific branding color configuration options.
+
+.From there you can:
+* Click on the *variable's* rounded color input, which will pop up a modal window with a color picker.
+* Pick a color then click the btn:[Save and close ] button to save your input variable color changes.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "corteza-studio/choose-color.png",
+    "alias": "corteza-studio-choose-color",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "focus": {
+    "x": 577,
+    "y": 490,
+    "w": 1087,
+    "h": 579
+  },
+  "annotations": [{
+    "kind": "box-note",
+    "x": 375,
+    "y": 212,
+    "w": 1130,
+    "h": 733
+  }]
+}
+----
+
+After making your branding color changes, click the btn:[Submit] button. This action will save your input changes and automatically refresh your admin web application with the updated branding color.
+
+=== Custom CSS Editor
+
+Under menu:User Interface[Settings] Custom CSS section.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "corteza-studio/custom-css.png",
+    "alias": "corteza-studio-custom-css-editor",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "focus": {
+    "x": 577,
+    "y": 490,
+    "w": 1087,
+    "h": 579
+  },
+  "annotations": [{
+    "kind": "box-note",
+    "x": 946,
+    "y": 375,
+    "w": 1130,
+    "h": 473
+  }]
+}
+----
+
+In this section, you can add your stylesheet code that specifically targets your web application(s). Afterward, simply click the btn:[Submit] button to save your input changes. This action will save your input changes and automatically refresh your web application with the updated styling.
+
+[NOTE]
+====
+Custom CSS editor supports both SASS and CSS syntax enabling you to customize your {PRODUCT_NAME} web application(s) styling using your preferred stylesheet language.
+====
```

Comment:
Personally, I'd move this just below `Under menu:User Interface[Settings] Custom CSS section`.
Primarily because it'd be neat to point it out earlier as well as me not liking to have multiple admonitions next to each other (these important, note, ... boxes).